v2/apiserver/internal/core/rest/logs_endpoints.go

```diff
@@ -47,6 +52,34 @@ func (l *LogsEndpoints) stream(
 		Follow: follow,
 	}
 
+	var lastEventID int64
+	// SSE has support for resuming where you left off after a
+	// disconnect/reconnect. Formally, I (krancour) believe the specification says
+	// clients can echo the ID of the last message received in a Last-Event-ID
+	// header, but I've encountered clients that use a lastEventId query parameter
+	// instead. Invoking Postel's principle, we'll support both.
+	if sse {
+		lastEventIDStr := r.Header.Get("Last-Event-ID")
+		if lastEventIDStr == "" {
+			lastEventIDStr = r.URL.Query().Get("lastEventId")
+		}
+		if lastEventIDStr != "" {
+			var err error
+			lastEventID, err = strconv.ParseInt(lastEventIDStr, 10, 64)
+			if err != nil {
+				restmachinery.WriteAPIResponse(
+					w,
+					http.StatusBadRequest,
+					&meta.ErrBadRequest{
+						Reason: "Value of \"Last-Event-ID\" header was not parseable as " +
```

Comment:
Technically, the value could also come from the corresponding url query... not sure if we should reword to be more generic ("Value of the last event ID was not parseable...") or other.  In any case, perhaps not a big deal.

Reply:
Good thought. I think I wrote that error message _before_ becoming more forgiving about how last id was sent to us.

I'll reword as you suggest.